packages/react-dom/src/client/ReactDOMFiberComponent.js

```diff
@@ -858,6 +859,7 @@ export function diffHydratedProperties(
       break;
     case 'img':
     case 'image':
+    case 'link':
       trapBubbledEvent('topError', 'error', domElement);
       trapBubbledEvent('topLoad', 'load', domElement);
```

Comment:
For some background:

I imagine we'll hit this again in the future as new HTML elements support onLoad/onError (though that's probably rare). As far as I understand it, we need to do this eager attachment because `error` and `load` to not bubble, so you could technically do:

```javascript
<div onLoad={() => alert('load!')>
  <img src="image.jpg" />
</div>
```

I sort of wonder if we should just respect the existing DOM behavior and avoid needing to eagerly attach these listeners. I'll write up an RFC for it!